code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -802,25 +779,8 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 
 
 
-/datum/clockwork_scripture/create_object/vitality_matrix
```

Comment:
I've got a slightly better idea for this than just removing it that'll make it generally more actually usable; Letting it consume non-servant corpses for vitality.